Assets/MixedRealityToolkit/Utilities/Lines/Renderers/MixedRealityLineRenderer.cs

```diff
@@ -57,6 +57,13 @@ private void OnEnable()
                 lineMaterial = lineRenderer.sharedMaterial;
             }
 
+            // mafinc - Start the line renderer off disabled (invisible), we'll enable it
```

Comment:
This looks like it needs to be changed to an issue number. I think mafinc is an alias.